flake.nix

```
@@ -14,7 +14,7 @@
   };
   inputs.nix.url = "github:domenkozar/nix/relaxed-flakes";
 
-  outputs = { self, nixpkgs, pre-commit-hooks, nix, ... }:
+  outputs = { self, nixpkgs, pre-commit-hooks, nix, ... }@inputs:
```

```suggestion
  outputs = { self, nixpkgs, pre-commit-hooks, nix, ... }:
```
This is not needed :)